gulpfile.js

```diff
@@ -1,16 +1,26 @@
+const babel = require('babelify');
 const gulp = require('gulp');
 const browserSync = require('browser-sync');
+const browserify = require('browserify');
```

Comment:
@andela-oowonikoko Hound is right here. Add bebelify to your dependencies not devDependencies. The implication of that is that babelify will only work on your dev environment if it is under devDependencies.